<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_almacenes</class>
 <widget class="QMainWindow" name="form_almacenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1241</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCentral">
     <attribute name="title">
      <string>Almacén Central</string>
     </attribute>
     <widget class="QMdiArea" name="mdiCentral">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>791</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_compra">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Compra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_moveToLocal">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Mover a Almacén Local</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_extraer">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Extraer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_newProduct">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Nuevo Producto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_eliminarProduct">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>40</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Producto</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_transactions">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Transacciones</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>20</y>
        <width>391</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_Centralsearch"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="centralTable">
         <column>
          <property name="text">
           <string>Producto</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cantidad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Precio x Unidad</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLocal">
     <attribute name="title">
      <string>Almacén Local</string>
     </attribute>
     <widget class="QMdiArea" name="mdiLocal">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>741</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_moveToCentral">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Mover a Almacén Central</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_localTransactions">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Transacciones</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>391</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_localSearch"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="localTable">
         <column>
          <property name="text">
           <string>Producto</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cantidad</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Precio</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
